### PR TITLE
Instead of "Hello", use {{ page.title }} for the <title> tag.

### PR DESCRIPTION
**Pull request by pgib**

### CODE REVIEW

**Comment by pgib** [comment]
[ch31078]

File: lib/jekyll/attendease_plugin/event_layout_page.rb [hunk 1]
```diff
@@ -16,12 +16,12 @@ def initialize(site, base, dir, name, base_layout, title_prefix)
         self.data['layout'] = base_layout
         self.data['base_layout'] = base_layout
 
-        title_prefix = 'Hello' if title_prefix == 'Layout' # Use "Hello" for the title of the main layout page.
+        title_prefix = '{{ page.title }}' if title_prefix == 'Layout' # Use "Hello" for the title of the main layout page.
```
Comment (Canuckaholic) [inline lib/jekyll/attendease_plugin/event_layout_page.rb]: Comment isn't accurate anymore. Also, what will be shown if `{{ page.title }}` isn't set? Can we have a fallback? Maybe something better than "Hello" though.

Comment (pgib) [reply lib/jekyll/attendease_plugin/event_layout_page.rb]: @Canuckaholic setting the title is now the responsibility on the Attendease side, and you can decide what that should be in `rendered_with_event_theme`.

Comment (Canuckaholic) [reply lib/jekyll/attendease_plugin/event_layout_page.rb]: Ok. In that case we just need to update the comment (so it doesn't confuse people of the future).

`# Use "Hello" for the title of the main layout page.`

**Pull request merged**